boxoffice/messageclient.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from baseframe import _
+
+from . import app, rq
+from .models import Order
+
+
+@rq.job('boxoffice')
+def send_telegram_message(order_id):
+    with app.test_request_context():
+        order = Order.query.get(order_id)
```

Comment:
`order` may be `None`, so add `if order is not None:` block for the rest of the code.